vulnfeeds/vulns/vulns.go

```diff
@@ -229,17 +229,25 @@ func extractAliases(id string, cve cves.CVE) []string {
 	return aliases
 }
 
-func FromCVE(id string, cve cves.CVEItem, pkg, ecosystem, purl, versionType string, validVersions []string) (*Vulnerability, []string) {
+type PackageInfo struct {
+	PkgName      string
```

Comment:
Can you add struct tags per https://pkg.go.dev/encoding/json#Marshal to make these use `snake_case` instead to match the OSV schema's casing?